addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -46,11 +46,11 @@
                 <div class="d-flex gap-2 align-items-baseline">
                     <h3 t-attf-class="col-lg-10 mb-0" t-out="question.name"/>
                     <div class="col d-flex justify-content-end align-items-center">
-                        <i t-if="question.state == 'close'" class="fa fa-lock ms-2 fs-4" title="Closed" data-bs-toggle="tooltip" data-bs-placement="top"/>
-                        <span t-elif="not question.active" class="badge bg-danger">
-                            <t t-if="question.state!='offensive'">Deleted</t>
-                            <t t-if="question.state=='offensive'">Offensive</t>
-                            <t t-if="question.state=='offensive' and question.closed_reason_id">
+                        <i t-if="question.state == 'close'" class="fa fa-lock ms-2 fs-4" title="Closed" data-bs-toggle="tooltip" data-bs-placement="top"></i>
```

Comment:
why this change ?

Reply:
not relevant anymore, so changing it back to how it was, thanks for pointing it out :)